### DISCUSSION TITLE
is this project abandon?

### DISCUSSION THREAD

**Discussion by ctwhome** [kimlimjustin/xplorer#287] [General]
it looks like the last release was made almost 2 years ago now

**Comment by in4matix** [comment]
What a damn pity !! Some nifty Dev should take over active development...

**Comment by kimlimjustin** [comment]
Hello guys, it's me here. Just here to clarify some things,

The progress toward this project is that since June, I've been working hard in refactoring the whole project to React-based project as the current memory management, efficiency, and beyond are really awful. 

Reasons why I just started it on June is that there are so many issues that can't really be fixed due to the issue of the dependence module itself. Tauri, as the foundation of this project, was in the beta version with a lot of issues that are unable to solve. Furthermore, the project stuck there due to inability of adding new features as Tauri doesn't really support it yet (some of the issue even remains today). One example is that the most important feature, drag and drop are unable to implemented as the results of [this feature request](https://github.com/tauri-apps/tauri/issues/2593) yet to be completed. I hope it's going to implemented soon.

Besides, lack of contributors, maintainers and fundings are also the main problems. As a student myself, I certainly have my own studies to focus itself, making the development pace to be very slow. Contributors and maintainers are actually expected to be here to help the growth of the project itself, but yeah, barely anyone wanted to help. Fundings as well, as the project management would need some costs to expand, it would require a lot of money, which is me myself couldn't afford as a student.

Despite all hardness, I still try my best to the refactoring process, and hope that issues addressed above could be solved really soon.

Discussions are welcomed :)

**Comment by in4matix** [comment]
Hello Mr Justin Maximillian Kimlim, really good to hear from you, and to hear that Xplorer is still being worked on!

I feel you brother, I have been there myself, and pretty much really still in the same boat. I am very new to development and coding, etc. Still learning as I go along. I would definitely like to help, but at this stage I will most likely mess things up more than anything else. I will start by sponsoring you. As they say every little bit helps...

Please keep us posted and I will try my best to spread the word.

Take care friend, hope to see some progress in the near future.

Thank you for all your hard work.

**Reply by kimlimjustin** [reply]
Thank you very much bro... probs further announcement of the development will be announced in several weeks.
